queries.sql

```diff
@@ -1,3 +1,46 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals
+WHERE name LIKE '%mon';
+
+SELECT * FROM animals
+WHERE date_of_birth >= '2016-01-01' AND date_of_birth <= '2019-12-31';
+
+SELECT * FROM animals
+WHERE neutered = 't' and escape_attempts < 3;
+
+SELECT date_of_birth FROM animals
+WHERE name IN ('Agumon', 'Pikachu');
+
+SELECT name, escape_attempts FROM animals
+WHERE weight_kg > 10.5;
+
+SELECT * FROM animals
+WHERE neutered = 't';
+
+SELECT * FROM animals
+WHERE name <> 'Gabumon';
+
+SELECT * FROM animals
+WHERE weight_kg >= 10.4 AND weight_kg <=17.3;
+
+
+-- transactions
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT weight_change;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO weight_change;
+COMMIT;
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT * FROM animals ORDER BY escape_attempts DESC LIMIT 1;
```

Comment:
- [ ] Here we have to find `Who escapes the most, neutered or not neutered animals?`.
```sql
SELECT neutered, AVG(escape_attempts) FROM animals
   GROUP BY neutered;
```